### ISSUE TITLE
experiment reproduction

### ISSUE THREAD

**Issue by hooxizz** [mr-eggplant/SAR#1]
Hi, there.

Thanks for sharing this great work.

I download the code and try to reproduce the results in table 4. However, the results is lower than reported in paper.

due to gpu limitation, I just run two corruption.

I just run:
`python3 main.py --data_corruption ./data/ImageNet-C \`
`--exp_type bs1 \`
`--method sar \`
`--model resnet50_gn_timm \`
`--output ./output/`

For Resnet50-GN results in table 4. My result is： 
`2023-03-01 18:39:05,186 INFO    : Result under shot_noise. The adaptation accuracy of SAR is top1: 22.83800 and top5: 40.58400`
`2023-03-01 18:39:05,198 INFO    : acc1s are [20.255998611450195, 22.83799934387207]`
`2023-03-01 18:39:05,198 INFO    : acc5s are [37.077999114990234, 40.58399963378906]`

For VitBase-LN results in table 4. My result is： 
`2023-03-01 18:24:27,754 INFO    : Result under shot_noise. The adaptation accuracy of SAR is top1: 25.73800 and top5: 44.81800`
`2023-03-01 18:24:27,820 INFO    : acc1s are [32.34600067138672, 25.737998962402344]`
`2023-03-01 18:24:27,820 INFO    : acc5s are [53.21399688720703, 44.817996978759766]`

Did I miss something? Looking forward to your reply.

**Comment by mr-eggplant** [comment]
Hi,

Thanks for your interest and for finding this bug.

The reason for this issue is that the hyper-parameter [learning rate] was incorrectly set in the previous code for BS=1 experiments. For further details, please refer to Appendix C.2 of the paper.

I have added lines 205-207 in _main.py._ Now you can check and verify the BS=1 results in Table 4 again.

Below is the result that I reproduced with ResNet-50-GN and BS=1 (same as the paper):

2023-03-01 21:09:46,089 INFO    : Result under gaussian_noise. The adaptation accuracy of SAR is top1: 23.79600 and top5: 42.32000
2023-03-01 21:09:46,090 INFO    : acc1s are [23.79599952697754]
2023-03-01 21:09:46,090 INFO    : acc5s are [42.31999969482422]

**Comment by hooxizz** [comment]
great! Thanks for your kind reply
